repose-aggregator/core/core-lib/src/main/java/org/openrepose/powerfilter/intrafilterLogging/RequestLog.java

```diff
@@ -20,17 +20,20 @@
 package org.openrepose.powerfilter.intrafilterLogging;
 
 import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang3.StringUtils;
 import org.joda.time.DateTime;
 import org.openrepose.commons.utils.servlet.http.MutableHttpServletRequest;
+import org.openrepose.core.systemmodel.Filter;
 import org.openrepose.powerfilter.filtercontext.FilterContext;
 
 import javax.servlet.ServletInputStream;
 import java.io.IOException;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.LinkedHashMap;
-import java.util.List;
+import java.util.*;
 
+/**
+ * POJO that is used to log details about a response when the log level is set to TRACE.
```

Comment:
Potential inaccuracy*

Considering the constructor contains the logic for what gets logged (i.e. what goes into the fields), I thought the expected log level was relevant here.  I can change "when the log level is set to TRACE" to "(may not always be logged due to log level)" in order to decrease the likeliness of the documentation becoming stale.